### PR TITLE
[docs] update readme

### PR DESCRIPTION
**Pull request by abiogenesis-now**
@hausdorff PTAL, thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
+Specifically, ksonnet allows you to:
+* *Reuse* common manifest patterns (within your app or from external libraries)
+* *Customize* manifests directly with powerful object concatenation syntax
+* *Deploy* app manifests to multiple environments
+* *Diff* across environments to compare two running versions of your app
+* *Version control* the entire state of your app manifests
 
-Status: Basic functionality works, and the tool is usable.  The focus
-now is on clearer error reporting and advanced features.
+*STATUS: Development is ongoing--this tool is pre-alpha.*
```
Comment (seh) [inline README.md]: - [ ] Nit: Use an em dash there.  
  s/--/—/

File: README.md [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
-infrastructure and reuse these powerful "templates" across many
-services, and then manage those templates as files in version control.
-The more complex your infrastructure is, the more you will gain from
-using kubecfg.
+Specifically, ksonnet allows you to:
+* *Reuse* common manifest patterns (within your app or from external libraries)
+* *Customize* manifests directly with powerful object concatenation syntax
+* *Deploy* app manifests to multiple environments
+* *Diff* across environments to compare two running versions of your app
+* *Version control* the entire state of your app manifests
```
Comment (seh) [inline README.md]: - [ ] "Version control" is odd as a verb.  
  Consider instead a few alternatives:  
  > Track changes to the entire state of your app manifests in your VCS
  > Capture the entire state of your app manifests in your VCS
  > Use your VCS to specify the entire state of your app manifests

File: README.md [hunk 3]
```diff
@@ -3,16 +3,16 @@
 [![Build Status](https://travis-ci.org/ksonnet/ksonnet.svg?branch=master)](https://travis-ci.org/ksonnet/ksonnet)
 [![Go Report Card](https://goreportcard.com/badge/github.com/ksonnet/ksonnet)](https://goreportcard.com/report/github.com/ksonnet/ksonnet)
 
-A tool for managing Kubernetes resources as code.
+ksonnet provides a framework for you to write, share, and deploy Kubernetes application manifests. With its CLI, you can generate a complete application from scratch in only a few commands, or manage a complex system at scale.
```
Comment (seh) [inline README.md]: - [ ] I stumble over the "for you to" part on each reading.  
  Consider:  
  > _ksonnet_ is a framework for writing, sharing, and deploying ...

**Comment by jessicayuen** [comment]
What's here so far looks great.

I'd like to expand on it a little to think through what we want in this README before release. IMO It'd be nice to see the following "sections",

1. Core cases: More or less what's in the README currently for `apply`, `diff`, etc. on files supporting `jsonnet`. 
2. Prototypes (definition & command examples)
    - emphasize on how we are able to quickly create k8s objects
3. Environments (definition & command examples):
    - emphasize on how we are able to manage multiple target servers

**Comment by abiogenesis-now** [comment]
thanks @seh for the feedback--I've incorporated and squashed those changes. 

@jessicayuen that's a fair point--I made this PR because @hausdorff wanted an updated blurb, and agree that the current README is not complete as is. That being said, I am in favor of punting any major README changes until we've figured out the plans for the docs website---*unless we have active users of this repo that are not developing the tool*.

^ @hausdorff is that the case?

**Comment by hausdorff** [comment]
@abiogenesis-now @jessicayuen Completely agree with Jess, but we should ship this change now so that people stop IM'ing me to complain that it's confusing.

**Comment by jessicayuen** [comment]
Ok, that works for me -- just wanted to make sure we capture those cases
before release.

On Oct 24, 2017 5:47 PM, "Jessica Yao" <notifications@github.com> wrote:

thanks @seh <https://github.com/seh> for the feedback--I've incorporated
and squashed those changes.

@jessicayuen <https://github.com/jessicayuen> that's a fair point--I made
this PR because @hausdorff <https://github.com/hausdorff> wanted an updated
blurb, and agree that the current README is not complete as is. That being
said, I am in favor of punting any major README changes until we've figured
out the plans for the docs website---*unless we have active users of this
repo that are not developing the tool*.

^ @hausdorff <https://github.com/hausdorff> is that the case?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/ksonnet/ksonnet/pull/32#issuecomment-339179122>, or mute
the thread

**Pull request merged**